src/pages/es/reference/integrations-reference.md

```diff
@@ -191,18 +191,20 @@ Una copia de solo lectura de [Astro config](/es/reference/configuration-referenc
 
 **Tipo:** [`ViteDevServer`](https://vitejs.dev/guide/api-javascript.html#vitedevserver)
 
-Una instancia mutable del servidor de Vite utilizada en modo "dev" y "vista previa". Por ejemplo, esto es [utilizado por nuestra integración Partytown](https://github.com/withastro/astro/tree/main/packages/integrations/partytown) para inyectar el servidor Partytown como middleware:
+Una instancia mutable del servidor de Vite utilizada en modo "dev" y "vista previa". Por ejemplo, esto es [utilizado por nuestra integración Partytown](/es/guides/integrations-guide/partytown/) para inyectar el servidor Partytown como middleware:
 
 ```js
-import
-
-'astro:server:setup': ({ server }) => {
-  server.middlewares.use(
-    partytownServer(partytownLibDirectory, {
-      mount: '/~partytown',
-      ...
-    })
-  );
+export default {
+  name: 'partytown'
+  hooks: {
+    'astro:server:setup': ({ server }) => {
+      server.middlewares.use(
+        function middleware(req, res, next) {
+          // handle requests
```

Comment:
I haven't noticed at first, but here's a small comment needing translation.

Reply:
> Thanks so much for this @kevinzunigacuellar and @alfredogonzalezmartinez! ❤️ Hope you don't mind I updated the translation to include some recent code and link related changes from [f449538](https://github.com/withastro/docs/commit/f449538343d16b15fb5d07469224504665437625) & [23ea009](https://github.com/withastro/docs/commit/23ea009f28a75df021b9fbf37ce18e741a75c07c).
> 
> I'll wait for a LGTM just to be sure I didn't break anything and then merge it! 🙌

Yes, I was going to give it a final read since this PR has almost two weeks. Thank you for referencing commits I will update those changes right now